lighthouse-core/test/fixtures/traces/frame-metrics-m89.devtools.log.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
same trace as before? :)

Reply:
That one didn't have any layout shifts. Maybe replace the previous trace with this and use for both?

Reply:
you read my mind :D

Reply:
I'm thinking merge https://github.com/GoogleChrome/lighthouse/pull/11701 and make the change here, sound good?